src/core/stdcpp/string.d

```diff
@@ -0,0 +1,337 @@
+﻿/**
+ * D header file for interaction with C++ std::string.
+ *
+ * Copyright: Copyright Guillaume Chatelet 2014 - 2015.
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors:   Guillaume Chatelet
+ * Source:    $(DRUNTIMESRC core/stdcpp/string.d)
+ */
+
+module core.stdcpp.string;
+
+/**
+ * Because of broken name mangling of extern C++, this file is only available
+ * for Linux platform right now. Name mangling is hardcoded with pragmas.
+ */
+version(linux):
+pragma(lib, "stdc++");
```

Comment:
because of the `alias basic_string!char std_string`, the struct gets instantiated and linker asks for `c_str` and `size`. I can hide the aliases behind a version, what do you think about `CRuntime_Glibcpp` ? Any other suggestion ? I also dislike this explicit dependency on stdc++.